app/server/dao/src/main/resources/io/syndesis/server/dao/deployment.json

```diff
@@ -432,8 +432,8 @@
         "specification": {
           "componentProperty": true,
           "deprecated": false,
-          "description": "Upload the OpenAPI 2.0 file for your Custom API Client Connector. Custom API's are RESTful APIs and can be hosted anywhere, as long as a well-documented API specification is available and conforms to OpenAPI standard.",
-          "displayName": "Specification",
+          "description": "Upload the OpenAPI 2.0 file for your Custom API Client Connector. Custom API's are RESTful APIs and can be hosted anywhere, as long as a well-documented API is available and conforms to OpenAPI specification.",
```

Comment:
Is this meant to be `specification` or `document`?

Reply:
Nevermind, I misread, the document should conform to the OpenAPI specification, so this is right! Thanks @TovaCohen !